src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj

```diff
@@ -630,6 +630,12 @@
     <None Include="ConverterTestData\StaticDriverVerifier\cancelspinlock_bug1.tt.sarif">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="ConverterTestData\PyLint\PyLint-01.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Why does GitHub show `PyLint-01.json` as a binary file? Is it in some strange text encoding?